app/views/article/show.html.slim

```diff
@@ -7,7 +7,8 @@
     -if description.present?
       .big ==description
     -else
-      i =link_to 'Edit the description in the settings tab.', { :controller => 'article', :action => 'edit', :article_id => @article.id }
```

Comment:
A guest should not be able to edit the description.  Transcription and translation is really all we want them to be able to do.